api/client.go

```diff
@@ -99,6 +103,10 @@ func NewClient(cfg Config) (Client, error) {
 	}, nil
 }
 
+func ClientCloseIdler(cl Client) {
+	cl.(closeIdler).CloseIdleConnections()
+}
+
```

Comment:
```suggestion
func ClientCloseIdler(cl Client) {
	cl.(closeIdler).CloseIdleConnections()
}
```

No need for that, anyone can do cl.(CloseIdler).CloseIdleConnection()

Reply:
done